resources/views/components/cd/form/README.md

```diff
@@ -0,0 +1,181 @@
+
+```
+    <x-cd.form.checkbox-inline name="inline"  label="Select me" description="description" wire:model.live="subscribe"/>
+```
+## Checkboxes
+
+The checkboxes component implements a set of related checkboxes which are bound to one Livewire variable. 
+An array of options define the set of checkboxes as demonstrated here.
+
+```
+    $this->checkboxoptions = [
+      [ 'value' => "tomato", "label" => "Tomato"],
+      [ 'value' => "lettuce", "label" => "Lettuce"],
+      [ 'value' => "pickle", "label" => "Pickle"], 
+      [ 'value' => "onion", "label" => "Onion"], 
+    ]; 
+```
+
+```
+    <x-cd.form.checkboxes :checkboxes="$checkboxoptions" wire:model.live="toppings" label="Topping Choices"/>
+```
+
+## Special text inputs
+
+The CD Framework provides styling for several special-purpose text inputs. These may be selected
+using the `type` attribute, as in the following examples:
+
+
+```
+    <x-cd.form.text type="search" label="Search" wire:model.live="search"/>
+    <x-cd.form.text type="telephone" label="Telephone" wire:model.live="telephone"/>
+    <x-cd.form.text type="url" label="URL" wire:model.live="url"/>
+    <x-cd.form.text type="email" label="Email" wire:model.live="email"/>
+    <x-cd.form.text type="password" label="Password" wire:model.live="password"/>
+    <x-cd.form.text type="number" label="Number" wire:model.live="number"/>
+    <x-cd.form.text type="datetime" label="Datetime" wire:model.live="datetime"/>
+    <x-cd.form.text type="datetimelocal" label="Datetime Local" wire:model.live="datetimelocal"/>
+    <x-cd.form.text type="date" label="Date" wire:model.live="date"/>
+    <x-cd.form.text type="month" label="Month" wire:model.live="month"/>
+    <x-cd.form.text type="week" label="Week" wire:model.live="week"/>
+    <x-cd.form.text type="time" label="Time" wire:model.live="time"/>
+```
+## Range
+
+A range input rendered as a slider may be specified using this special text input which requires
+`max` and `min` attributes.
+
+```
+    <x-cd.form.text type="range" required="1" label="Range" min="1" max="10" wire:model.live="range"/>
+```       
+
+## File
+
+The file selector input is also a variant of the text input.
+```
+    <x-cd.form.text type="file" required="1" label="File" wire:model.live="file"/>
+```
+
+## Color
+
+The color picker is also a variant of the text input.  Take care to initialize the value of this
+input to a valid `value`, which is a hash (#) character followed by six hexadecimal digits. 
+```
+        <x-cd.form.text type="color" label="Color" wire:model.live="color" 
+           value="#ff0000"/>
+```
+        
+# Radio Buttons
+
+The `radios` component implements a set of related radio buttons defined by an array of options, as demonstrated below.</p>
+
+```
+    <x-cd.form.radios label="Radios" wire:model.live="radios" :radiobuttons="$radiooptions" />
+```
+Using `inline=1` alows the radio buttons to be displayed inline with flexbox. 
```

Comment:
I couldn't find evidence of this feature actually existing, but it would be great to have!